pyproject.toml

```diff
@@ -19,7 +19,8 @@ dependencies = [
     'gwcs >=0.18.1',
     'jsonschema >=4.0.1',
     'numpy >=1.22',
-    'photutils >=1.6.0',
+    # temporary pin to photutils dev until >1.8.0 is available:
+    'photutils @ git+https://github.com/astropy/photutils.git',
```

Comment:
The next release of photutils may be out within a week or so, and I will update this pin when it's available.